### PR TITLE
Customized all readme file

### PR DESCRIPTION
**Pull request by RuriEnciso**
Hello, I make corrections from this first PR:   https://github.com/RuriEnciso/Hello-World/pull/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,279 +1,42 @@
...
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
```
Comment (shella12) [inline README.md]: - [ ] Great job 👍 using the [Microverse template](https://github.com/microverseinc/readme-template) for your README and customizing it to reflect your project as suggested by the previous reviewer.  But I can see 👀 you have removed the content table 🗒️  and missed a lot of sections from the content table. Please make sure you add them into your README as it is a [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) of the project to use the [template](https://github.com/microverseinc/readme-template) for README and include all sections from the template. The only section that is OPTIONAL to add is FAQ. 
- [ ] Please 🙏  make sure to customize each section to reflect your project
- [ ] Also make sure to remove comments and _**Note**_ after you are done customizing ☺️

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @RuriEnciso , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. You are almost there...



## Highlights

- Added meaningful branch name ✅ 
- Made pull request descriptive ✅ 

 ## Required Changes ♻️

- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (RuriEnciso) [reply README.md]: hello @shella12.

Comment (RuriEnciso) [reply README.md]: I´m sorry but, the content of my first project is very short, just a main title in html and its font color in css.
I only used html and css so I don't have enough material to fill all the sections of the microverse template.
Also, in the 7th line of the microverse template it says this: "Modify this file to match your project and remove sections that don't apply." what can I do?

Comment (shella12) [reply README.md]: Hey @RuriEnciso I understand that it's difficult 😅  to come up with something like the Future features or key features for such a small project. Microverse requires students to stick to the template as practice for future projects. I believe you can always come up with some content to fill the required section lets say for example Future feature includes Goodbye Microverse or Colors and Fonts lets say. I hope you understand. If you still need further explanation feel free to reach out ☺️

Comment (RuriEnciso) [reply README.md]: OKay, I will try. thank You.

**Review by joel-kalema — APPROVED** [review]
Hi @RuriEnciso 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

The project is running successfully 👍 
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**